### PR TITLE
Fix Sonar in CI to get coverage back

### PR DESCRIPTION
**Pull request by Hartorn**
## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [ ] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

**Comment by mattbit** [comment]
@Hartorn 

> **Last analysis has warnings**
> The following error(s) occurred while trying to import coverage report:
> Invalid directory path in 'source' element: /home/runner/work/giskard/giskard/giskard%nCannot resolve 149 file paths, ignoring coverage measures for those files%nCannot resolve the file path '__init__.py' of the coverage report, the file does not exist in all 'source'.

File: giskard/core/suite.py
Comment (mattbit) [inline giskard/core/suite.py:1]: Why this is here?

**Review by mattbit — CHANGES_REQUESTED** [review]
I think modifications of `suite.py` were erroneously included.

File: giskard/core/suite.py
Comment (Hartorn) [reply giskard/core/suite.py:1]: I think it's because of main was updated with some stuff, I get some conflict

**Comment by Hartorn** [comment]
@mattbit did you get notified before with PR was not in draft anymore ? or did you saw it in pull requests ?

**Review by mattbit — APPROVED** [review]

**Pull request merged**